features/fixtures/business_readiness_filtered_results.json

```diff
@@ -58,7 +58,7 @@
             "other-eu"
           ],
           "index": "detailed",
-          "es_score": null,
+          "es_score": 10000.0,
```

Comment:
Do both fixture files need to have es_scores? For some searches (A-Z etc) the es_score is always null.

Reply:
I have removed es_score out of business_readiness_results.json

Reply:
I think you should leave the score as `null` rather than taking it out completely as that's what the data from rummager looks like.